### ISSUE TITLE
Feedback Collection

### ISSUE THREAD

**Issue by Fenyx4** [cambragol/advent-of-the-trinity#56]
I just gathered a bunch of small feedback in this one issue so it wouldn't spam things. Let me know if you would prefer I split them up.

1. I like the new Mandrake location. I recently saw someone's first playthrough of the game and (after failing with the single spot because they didn't know the two moons) down there was where they spent a lot of time searching.

1. One of the new characters reference Data from TNG! Loved it!

1. The fire graphics in the opening are awesome! I love the reflection! Very impressive!

1. When entering battle the way the screen splits is cool. Having a transition there is really nice.
Only drawback is that it does take a little while to complete so when fighting 3 things in a row it can drag on a bit.

1. Asking how many of a mixture you want mixed for spells. I think you know my feelings on this. IT IS GREAT!

1. Changing the fountain on the top level of Despise to no longer be a heal fountain. Ooo... Evil. I like it. :)

1. Poison and Sleep. YOU CAN BE BOTH POISONED AND SLEPT! Nice fix for a big exploit!!!

**Comment by cambragol** [comment]
Feedback in this form is great. There are a lot of issues, so gathering general stuff in one place is good.

The old spot for the Mandrake was just a liiiiiittle obvious. This new spot holds true to the original clue. There are other, newer spots as well! For Nightshade too!

That is NOT Data from TNG. It is another Data, from the greatest kids movie ever! And he is part of special, pop-culture quest (the only one), which hearkens back to the out-of-character NPC in the original, Short-round.

The fire graphics took some time. Glad it caught your eye. Went through many evolutions too. The whole quest story in the 'View' also took some serious work. It is all programmed in ASM data. So a real pain to decipher, then recode 5x as large as the original.

The screen split was thrown in to prevent people entering an encounter, then zooming out of it without realizing they had entered it. Now that I have added the query for leaving towns, 'Dost thou wish to leave', the split screen is totally superfluous. We can keep it if you guys like, or remove it and save a bit of code (very little).

Mixing multiple spells. Yup. Its good. I still remember mixing 99 tremor the old way, thinking, why do I have to do this one at a time!?

I searched the internet for cheats/walkthroughs that described exploits, then tried to fix them.  Some other stuff was done to top level of Despise, and Hythloth was hit up too (for orbs). If you know of any other obvious gamey/cheaty stuff, please inform, as I would like to polish such 'expoits' out, well, because they are exploits.

Sleep and Poison! Yes, that took a bunch of tweaking....a LOT actually, but it works now. Please look for any bugs in that one, as it was spread over a lot of places..camping, inns, effects of spells, combat...its everywhere. I THINK I got all the possible issues with it, but hey, you can't be perfect.

**Comment by Fenyx4** [comment]
There's more Mandrake and Nightshade? Interesting... Interesting indeed...

It isn't Data from TNG. Huh, I'll have to try following that thread.

Exploits... Lemme see. I've been considering doing a speedrun of U4 so I know a few of them.
Getting gold and reagents by grabbing the chests from Castle Britannia and robbing the herb person blind which tanks your karma but then you just grinding to get the karma back up. Grinding compassion by giving to a beggar, pass for a few rounds and then give again. Grinding Humil by talking to the same guy over and over. Spirituality by talking to Hawkwind. And Honesty, Justice and Honor by giving the proper amount to an herb woman (but only buy 1 garlic at a time). Sacrifice by giving blood.

Only playing with one character. People do this to keep down brain fatigue from handling 8 character. Your active player feature helps a lot with the brain fatigue problem and the increased difficulty gives a lot of incentive to have more characters.

After becoming Avatar you could get experience and gold by getting the Mystic Armor, selling it and getting it again.

Not terribly useful but you can override energy fields with a different type (except for lightning). So if you don't want to get slept you could override it with a poison field. But the SMS speedrunner is planning on using this since Energy Field is less mana and he spends the entire Abyss poisoned to avoid sleep.

**Comment by cambragol** [comment]
Whaaat!? You don't immediately know the Data reference with an 80s kid movie reference and connection to Short Round hint! Shock. Shock I tell thee! 

You might need to review some 80s movies that came out in 1985 if you want to get into that thread.

I think I have covered all the 'exploits' you mentioned, though perhaps not sufficiently for Compassion and Humility? I added cool-downs to most of those, which shouldn't be avoidable by just passing a few turns. I think.

The rest are most certainly covered, though cheating the blind herbalist could be improved. Maybe make her only get fooled by money within a range. I.e. if you give her one gold for 99 Black Pearl, she notices. Or..boohoohahaa...make each store have limited supplies!
